### PR TITLE
Feature/req1 6

### PR DESCRIPTION
**Pull request by kouissar**
just adding server stuff to the mix

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ function toggleLog () {
 }
 
 function init (slide) {
-  addResponse(slide.id, { 'appVersion': window.navigator['appVersion'] })
+  // define test start time and add to response stack
+  var d = new Date()
```
Comment (skatulich) [inline js/main.js]: please give variables meaningful names.

File: js/main.js [hunk 2]
```diff
@@ -150,6 +156,64 @@ function submit_slide(slide){
   slide.appendChild(element)
 }
 
+function submit_jira(slide){
+  var tester = document.getElementById('name').value
+  var env = JSON.stringify(window.DATA['responses'].init.user_environment)
+  var summary = env + ', Tester: ' + tester 
+  var uf = document.getElementById('feedback').value
```
Comment (skatulich) [inline js/main.js]: please give variables meaningful names.

**Review by skatulich — DISMISSED** [review]
Please address comments, but nothing major or keeping it from approval.

File: js/main.js (hunk 1 above)
Comment (kouissar) [reply js/main.js]: OK will do Steve. Thanks

File: package.json [hunk 3]
```diff
@@ -45,11 +45,14 @@
     "grunt-sass": "^2.0.0",
     "grunt-zip": "~0.17.1",
     "mustache": "^2.3.0",
+    "nodemon": "^1.18.5",
     "socket.io": "^1.7.3"
   },
-  "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
-    "ua-parser-js": "^0.7.19"
```
Comment (kemaric) [inline package.json]: You took out this dependency that's needed.

File: js/main.js (hunk 2 above)
Comment (kemaric) [reply js/main.js]: @kouissar Be sure to update these variables after we have this merged moved into the src directory.

Comment (kouissar) [reply js/main.js]: sure will do!

**Review by kemaric — DISMISSED** [review]
LGTM

**Review by skatulich — APPROVED** [review]
Made one more variable name change.

**Review by kemaric — APPROVED** [review]
lets get this merged

**Pull request merged**